2D Navigation Demo/navigation.gd

```diff
@@ -36,19 +39,24 @@ class car:
 		self._position_track = position_track
 		self._speed = speed
 
-func _process(delta): #delta is FPS, which is automaticallly adjusted by the system
-					  #depending on state of the computer at a given time
-	var walk_distance = character_speed * delta #this thus gives pixels/frame
+func _physics_process(delta):  # Delta = 1/60
+	var walk_distance = character_speed * delta
 	move_along_path(walk_distance)
 	
 	if Input.is_key_pressed(KEY_SPACE):	
-		#If timer not started  --> Start timer!
-		print(OS.get_ticks_msec())
+		if start_time == 0:
```

Comment:
Setting the start_time shouldn't be done in the update function, but in the function where the game is actually started. This way we don't have an if statement that is checked 60 times per second when it is only ever executed once.

Reply:
Further note: The game isn't started when pressing space for the firs time but when the track is "loaded" due to the fact we don't have a nice countdown screen yet.

The way this functionality is written now breaks the behavior when you have 2 players.